backend/server/compute/diffexp_generic.py

```diff
@@ -2,7 +2,7 @@
 from scipy import sparse, stats
 
 
-def diffexp_ttest(adaptor, maskA, maskB, top_n=8, diffexp_lfc_cutoff=0.01):
+def diffexp_ttest(adaptor, maskA, maskB, top_n=8, diffexp_lfc_cutoff=0.01, two_lists=False):
```

Comment:
As far as I can tell, this code is identical between czi_hosted/compute and server/compute/

Why is it not consolidated into backend/common?

Reply:
moved to backend/common